src/data/real/cau_seq.lean

```diff
@@ -603,6 +603,167 @@ theorem exists_lt (f : cau_seq α abs) : ∃ a : α, const a < f :=
 let ⟨a, h⟩ := (-f).exists_gt in ⟨-a, show pos _,
   by rwa [const_neg, sub_neg_eq_add, add_comm, ← sub_neg_eq_add]⟩
 
+-- so named to match `rat_add_continuous_lemma`
+theorem _root_.rat_sup_continuous_lemma
+  {ε : α} (ε0 : 0 < ε) : ∃ δ > 0, ∀ {a₁ a₂ b₁ b₂ : α},
+  abs (a₁ - b₁) < δ → abs (a₂ - b₂) < δ → abs (a₁ ⊔ a₂ - (b₁ ⊔ b₂)) < ε :=
+⟨ε, ε0, λ a₁ a₂ b₁ b₂ h₁ h₂,
+  (abs_max_sub_max_le_max _ _ _ _).trans_lt (max_lt h₁ h₂)⟩
+
+-- so named to match `rat_add_continuous_lemma`
+theorem _root_.rat_inf_continuous_lemma
+  {ε : α} (ε0 : 0 < ε) : ∃ δ > 0, ∀ {a₁ a₂ b₁ b₂ : α},
+  abs (a₁ - b₁) < δ → abs (a₂ - b₂) < δ → abs (a₁ ⊓ a₂ - (b₁ ⊓ b₂)) < ε :=
+⟨ε, ε0, λ a₁ a₂ b₁ b₂ h₁ h₂,
+  (abs_min_sub_min_le_max _ _ _ _).trans_lt (max_lt h₁ h₂)⟩
```

Comment:
You could strip the `exists` out and that would golf the proofs where you call them. This is not the same situation as `rat_add_continuous_lemma` because there `ε` and `δ` might actually be different.

Reply:
Sure, that's pretty reasonable. I was just blindly copying the API

Reply:
Done